webcomponents/tool/src/main/frontend/packages/sakai-picture-changer/src/SakaiPictureChanger.js

```diff
@@ -147,7 +147,7 @@ export class SakaiPictureChanger extends SakaiElement {
 
   _save() {
 
-    const base64 = this.cropper.getCroppedCanvas().toDataURL().replace(/^data:image\/(png|jpg);base64,/, "");
+    const base64 = this.cropper.getCroppedCanvas({ maxWidth: 600, maxHeight: 600 }).toDataURL("image/png").replace(/^data:image\/(png|jpg|webp);base64,/, "");
```

Comment:
It would be great to use webp instead, but I believe there are assumptions around using PNG only here, correct @adrianfish? If just PNG, we can take a look at that separately (later)